sample-apps/grpc-greeter-go/greeter-istio.yaml

```diff
@@ -0,0 +1,65 @@
+# Copyright 2019 Google LLC
```

Comment:
Move both YAML manifests into `manifests/` or `k8s/` directory? Thanks.

Reply:
Moved to manifests/